src/static/js/domline.js

```diff
@@ -61,6 +61,9 @@ domline.createDomLine = (nonEmpty, doesWrap, optBrowser, optDocument) => {
 
   if (document) {
     result.node = document.createElement('div');
+    // Set the ARIA role and label for JAWS compatibility. Do it here, because it is only needed and
+    // possible if we have a document and are working in a real browser context.
+    result.node.setAttribute('aria-live', 'assertive');
```

Comment:
Just to confirm: Is `assertive` the right value here? I'm not an ARIA expert, which is why I'm a bit concerned by [MDN's strong language](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#live_regions):

> `aria-live="assertive"` should only be used for time-sensitive/critical notifications that absolutely require the user's immediate attention.

Reply:
It is the right value we choose, we tested it and found it as a required value to make JAWS and NVDA work correctly. Before we checked "role=textbox" and "ariaLabel=Line". It was working also, but testing it in the school environment turned out that "assertive" is the best choice.
I am also not a expert in ARIA, it was just checking what makes the screenreader work correctly, and this depends on how the screenreader deals with that values.

Reply:
The changes you have committed above are OK, thank you! So it is good to let them flow into the merge as you are ready to do it.